test/integration/rmi_out_test.py

```diff
@@ -40,18 +40,35 @@ def test_pudl_engine(pudl_engine, table_name):
 def test_ppl_out(rmi_out, request):
     """Test generation of the EIA Plant Parts List."""
     clobber = not request.config.getoption("--cached-plant-parts-eia")
-    ppl = rmi_out.plant_parts_eia(clobber=clobber)
+    ppl = rmi_out.plant_parts_eia(
+        clobber=clobber,
+        pickle_train_connections=True,
+    )
     assert not ppl.empty
-    for ppl_df in ["plant_parts_eia", "gens_mega_eia", "true_grans_eia"]:
-        if ppl_df in rmi_out.pudl_out._dfs:
-            del rmi_out.pudl_out._dfs[ppl_df]
+    del ppl
+
+
+def test_ppl_distinct_out(rmi_out, request):
+    """Test generation of distinct EIA Plant Parts List."""
+    clobber = not request.config.getoption("--cached-plant-parts-eia-distinct")
+    ppl_distinct = rmi_out.plant_parts_eia_distinct(clobber=clobber)
+    assert not ppl_distinct.empty
+    del ppl_distinct
```

Comment:
is this the only place this `cached-plant-parts-eia-distinct` is being used?? i couldn't quite trace its need.

Reply:
Yep I added this flag just to be consistent with the rest of the tests which each have their own `--cached- ...` flag. But i could take it since the distinct-ing really doesn't take very long.

Reply:
For simplicity, decided to take out this flag for using the cached distinct PPE.